modules/@apostrophecms/search/index.js

```diff
@@ -50,8 +50,8 @@ module.exports = {
   extend: '@apostrophecms/page-type',
   options: {
     alias: 'search',
-    name: '@apostrophecms/search',
-    perPage: 10
+    perPage: 10,
+    label: 'apostrophe:searchLabel'
```

Comment:
No label made the editor sad. We should consider doing this for the home page as well. It's conceivable someone will park the home page and not allow editors to create new ones or change the type of the parked one.

Reply:
Makes sense.